examples/smoketest.sh

```diff
@@ -26,7 +46,7 @@ main() {
 
   for os in "${OSES[@]}"; do
     for i in $(seq 4); do
-      scenario$i $os
+      try scenario$i $os
```

Comment:
does this act like we expect in jenkins? or might we get some green builds that are actually broken if we `try` without `catch`?